lib/MadNLPGPU/src/interface.jl

```diff
@@ -222,13 +296,27 @@ end
     end
 end
 
+@kernel function ker_set_con_scale_sparse!(@Const(ptr), @Const(inds), con_scale, @Const(jac_I), @Const(jac_buffer))
```

Comment:
I would maybe add `con_scale` in first position as this is the array we are modifying inplace in this kernel. 
Also, maybe we should rename the kernel as `_set_con_scale_sparse_kernel!`?